ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -135,6 +135,10 @@ impl Blockstore {
             .db
             .delete_range_cf::<cf::SlotMeta>(&mut write_batch, from_slot, to_slot)
             .is_ok()
+            & self
+                .db
+                .delete_range_cf::<cf::BankHash>(&mut write_batch, from_slot, to_slot)
+                .is_ok()
```

Comment:
could you add similar code to `compact_storage` as par?: https://github.com/solana-labs/solana/blob/335ac82a1a78800f49b2079c32a4a623d5251b49/ledger/src/blockstore_db.rs#L354

I know it's already dead code. but for the sake of consistency. it won't hurt. :)

Reply:
good catch, totally missed this